kafka-connect-ftp/src/main/scala/com/datamountaineer/streamreactor/connect/ftp/source/FtpSourceConfig.scala

```diff
@@ -65,6 +72,7 @@ object FtpSourceConfig {
     .define(FileMaxAge, Type.STRING, Importance.HIGH, "ignore files older than this; ISO8601 duration")
     .define(MonitorTail, Type.LIST, "", Importance.HIGH, "comma separated lists of path:destinationtopic; tail of file is tracked")
     .define(MonitorUpdate, Type.LIST, "", Importance.HIGH, "comma separated lists of path:destinationtopic; whole file is tracked")
+    .define(MonitorUpdateSlice, Type.LIST, "", Importance.HIGH, "comma separated lists of path:destinationtopic; whole file is tracked and teeated as slices")
```

Comment:
typo: `teeated` => `treated`

I'm wondering if `slice` is the best way to name this new feature (both for its name and for its description). In my case, the name doesn't help me understand what it does and the description doesn't clarify it. Could we brainstorm this to try to find a meaningful name and description ?